### PR TITLE
Adds auto-merge to main branch.

### PR DESCRIPTION
**Pull request by artursouza**
Thank you for helping make the Dapr documentation better!

- [ ] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

Adds automerge cron to main repo.

How it works:
+ Issue must be approved by one of the maintainers.
+ Issue must be tagged with "automerge"
+ Cron runs every 10 minutes.
+ Merges up to 1 PR that is ready (merge button is green).
+ Then, it updates any other issue (approved and tagged) with the latest upstream changes.
+ Finally, lists which PRs it could not merge pr update.

## Issue reference

N/A

### CODE REVIEW

**Comment by artursouza** [comment]
After merging, please add Dapr-Bot as a writer to this repo.

File: .github/workflows/dapr-automerge.yml
Comment (mukundansundar) [inline .github/workflows/dapr-automerge.yml]: change maintainers.

Comment (artursouza) [reply .github/workflows/dapr-automerge.yml]: I thought I did. Odd. Let me see.

Comment (artursouza) [reply .github/workflows/dapr-automerge.yml]: I changed in the wrong repo :) That is why.

File: .github/workflows/dapr-automerge.yml
Comment (AaronCrawfis) [inline .github/workflows/dapr-automerge.yml]: Can this be the dapr\maintainers-docs group instead of individual users?

Comment (artursouza) [reply .github/workflows/dapr-automerge.yml]: Well. That would be a follow up. The script does not do that now.

Comment (artursouza) [reply .github/workflows/dapr-automerge.yml]: @AaronCrawfis Is this a blocker to merge this PR?

**Comment by artursouza** [comment]
@AaronCrawfis ?

File: .github/workflows/dapr-automerge.yml
Comment (AaronCrawfis) [reply .github/workflows/dapr-automerge.yml]: Nope, just behind on PRs 😄

Comment (AaronCrawfis) [inline .github/workflows/dapr-automerge.yml]:
```suggestion
          MAINTAINERS: AaronCrawfis,orizohar,msfussell
```

https://github.com/orgs/dapr/teams/maintainers-docs/members

**Review by AaronCrawfis — CHANGES_REQUESTED** [review]

**Review by AaronCrawfis — APPROVED** [review]

**Pull request merged**
